docs/app/components/navbar/navbar-demo.component.html

```diff
@@ -12,10 +12,15 @@ <h5>About</h5>
 
     <hc-tile>
         <h5>Example</h5>
-        <hc-navbar appIcon="./assets/CashmereAppLogo.svg" brandIcon="./assets/TriFlame.svg" user="Christine K." [homeUri]="undefined" [fixedTop]="false">
+        <hc-navbar appIcon="./assets/CashmereAppLogo.svg" brandIcon="./assets/TriFlame.svg" [homeUri]="undefined" [fixedTop]="false">
             <hc-app-switcher></hc-app-switcher>
-            <hc-navbar-link [active]="true" [uri]="undefined">Home</hc-navbar-link>
-            <hc-navbar-link [active]="false" [uri]="undefined">Buttons</hc-navbar-link>
+            <hc-navbar-link [active]="true" uri="undefined" linkText="Home"></hc-navbar-link>
```

Comment:
It is nice when the example in the markdown matches the demo. Could you update it with these changes?